examples/storage/bucket/resource-class.yaml

```diff
@@ -14,3 +14,25 @@ specTemplate:
   providerConfigRef:
     name: example
   reclaimPolicy: Delete
+---
+apiVersion: storage.aws.crossplane.io/v1alpha3
```

Comment:
You are welcome to just replace the existing one with the one with tags if you like :+1: